components/src/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
```

Comment:
Just asking for my own understanding: This file is served by `make serve-components`? And if I wanted to test out a new component, I'd add it to this file?

Reply:
Yes, that's right -- and now I see that I omitted this slightly relevant detail from the README. 😂 I'll add it.